aries-site/src/pages/templates/selector.mdx

```diff
@@ -0,0 +1,93 @@
+import { Notification } from 'grommet';
+import { ShareRounded } from 'grommet-icons';
+import { Example } from '../../layouts';
+import {
+  SimpleSelector,
+  QuickFilter,
+  SupportSelector,
+  TEMP,
+} from '../../examples';
+
+<Example
+  figma="TBD"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/SimpleSelector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/Selector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/SelectorGroup.js',
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-core/src/js/components/core/Selector"
+  width="100%"
+>
+  <SimpleSelector />
+</Example>
+
+## Use cases
+
+### Quick Filtering
+
+Enables users to filter data based on high-level visual options to narrow down datasets within tables and list. Making it quickly recognizable, based on specific criteria, enhancing efficiency and usability.
+
+<Example
+  figma="TBD"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/QuickFilter.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/Selector.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-core/src/js/components/core/Selector/SelectorGroup.js',
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-core/src/js/components/core/Selector"
+  width="100%"
+>
+  <QuickFilter />
+</Example>
+
+### Quick option selection
+
+Helps users quickly navigate through choices, ensuring they find the relevant options with ease.
+
+<Example
+  code=""
+  height={{ min: 'small' }}
+  width="100%"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/SupportSelector.js',
+  ]}
+  github="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selector/SupportSelector.js"
+>
+  <SupportSelector />
+</Example>
+
+## Anatomy
+
```

Comment:
There's no imagine for anatomy? It's hard to understand without a visual

Reply:
Totally get it. We have an image in Figma and something that's still a bit more TBD. Imagery will come later.

Reply:
anatomy to come..

Reply:
have to code it up